### DISCUSSION TITLE
Issue with Autogluon's "TabularPredictor" not Fitting "TabPFN" Model

### DISCUSSION THREAD

**Discussion by usamabuttar** [autogluon/autogluon#3327] [Q&A] [answered]
Hello, Autogluon community,

I hope you're doing well. I am currently facing an issue related to the integration of the "TabPFN" model with Autogluon's "TabularPredictor". Although I have successfully installed "TabPFN" and verified its standalone functionality, Autogluon's "TabularPredictor" does not fit the "TabPFN" model as expected.

When using Autogluon's "TabularPredictor" for fitting models on tabular data, it fits a range of models, but "TabPFN" does not seem to be included in the fitting process. I have gone through the necessary steps to ensure the correct installation of "TabPFN" and have confirmed its independent functionality.

I would greatly appreciate any guidance or suggestions on how to enable Autogluon's "TabularPredictor" to fit the "TabPFN" model. Is there a specific configuration or setting that needs to be adjusted? Are there any known issues or limitations regarding the integration of "TabPFN" with Autogluon?

If there are any additional steps, requirements, or considerations that need to be taken into account in order to include "TabPFN" in the fitting process, please provide instructions or documentation that can help resolve this issue.

Thank you for your time and assistance. I look forward to your response and resolving this matter.

Best regards,
Usama Buttar.

**Comment by shchur** [comment, marked as answer]
Hi, `TabPFN` model is not included in the default presets. To enable this model, you need to either pass `presets="experimental_extreme_quality"` or `hyperparameters={"TABPFN": {}}` to `TabularPredictor.fit`. Also, as far as I know, the model only works for classification problems.
